src/pages/index.md

```diff
@@ -3,60 +3,24 @@
 An effort to describe and quantify corporate contributions to open source.
 Our goal is to empower employees advocating for open source contributions to their employers. 💙
 
-[Find us on GitHub](https://github.com/OpenContributionsProject/opencontributionssite)!
+You can learn more about us in [About](./about).
 
-## Project Goals
+## Site Contents
 
-We plan on tackling two project focuses, in order of complexity:
+### Explainers
 
-- [**Explaining**](#explaining-open-source-contributions): writing convincing, data-backed resources for why open source contributions are in company’s best interests.
-- [**Quantifying**](#quantifying-corporate-open-source-contributions): creating a tracking service that spotlights open source contributions already being done by companies.
+Wondering what open source is, or why we should contribute to it?
+You're not alone.
+See our [Explainers](./explainers) to understand this incredible movement in software development.
 
-## Explaining Open Source Contributions
+### Guides
 
-_We_ know that donating money and time to open source is the correct and right thing for companies to do.
-But financial decision-makers act only when motivated to.
-They need real reasons -ideally metrics- and/or research-based- that demonstrate why potential investments will be net positive for the company.
+Interested in advocating for contributing to open source, or getting involved yourself?
+Let us help you.
+See our [Guides](./guides) for suggestions on how to go about making open source a part of your life.
```

Comment:
okay I love the set up and the explainers. This is all great. I just that "See our guides for suggestion on how to go about making open source a part of your life" is a little aggressive. I think we change this.

Reply:
Maybe something like
`Check out our [Guides](./guides) to learn how you can start your open source journey and make a positive impact on the world of software development`

Reply:
Haha, you're right 💯 . That snippet is way better.